privacyidea/lib/policy.py

```diff
@@ -2016,7 +2017,11 @@ def get_static_policy_definitions(scope=None):
                           "attribute, set this to '*'. Use '*' with CAUTION! "
                           "For more details, check the documentation."),
                 'mainmenu': [],
-                'group': GROUP.USER}
+                'group': GROUP.USER},
+            ACTION.TOKENINFO_API_EXCLUDE: {
+                'type': TYPE.STRING,
+                'desc': 'A comma-seperated list of token-info fields that get excluded in the web-ui'
```

Comment:
Tokeninfos with multiple values are whitespace-separated. See https://github.com/privacyidea/privacyidea/pull/2500/files. If you set unique=False in the Math above, then you immediately have all the values.

Maybe write something like

"A whitespace-separated list of tokeninfo fields which are not displayed to the user."

and the same for the admin.